packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -236,6 +258,8 @@ export default function NewCommentModal({
     ) {
       await sendComment(event);
     }
+
+    onMentionKeypress(event);
```

Comment:
I think this is better

```suggestion
else {
    onMentionKeypress(event);
}  
```